src/components/elementView/powerView/powerView.tsx

```diff
@@ -84,7 +84,7 @@ export class PowerView extends React.PureComponent<
       });
       axios
         .get(
-          API_ROOT + "api/power/get-state/" + this.props.asset!.id,
+          API_ROOT + "api/rack-power/status/" + this.props.asset!.id,
```

Comment:
we need to check that an asset is rack powered before calling this endpoint and more generally add the new power logic to the front end  (ex. if asset = blade, don't show rack power details, if asset= chassis, show chassis power details)